src/rf/apps/home/tests.py

```diff
@@ -48,12 +48,14 @@ def make_layer(self, layer_name, is_public=False):
                 {
                     'file_name': 'foo.png',
                     's3_uuid': 'a8098c1a-f86e-11da-bd1a-00112444be1e',
-                    'file_extension': 'png'
+                    'file_extension': 'png',
+                    'source_s3_bucket_key': ''
```

Comment:
I was thinking the same thing, but was afraid there was a reason we always have CharFields default to '' and not allow nulls.